README.md

```diff
@@ -0,0 +1,190 @@
+# xtm - a library for cross-thread messaging with event loop intergation
+
+The library provides the following facilities:
+
+# xtm_queue
+
+Opaque struct, that represent unidirectional, single-writer-single-reader queue
+implementation with eventloop integration ability.
+
+## xtm_queue_new
+
+Allocation function, used to create instance of struct xtm_queue and return
+its pointer or NULL in case of error. Accepts the queue size as input, which
+must be a power of two, and flags, defining library behavior.
+
+## xtm_queue_delete
+
+Deallocation function, used to free queue and close its internal fds.
+Returns 0 on success. Otherwise -1 with errno set appropriately (as in
+close(2), since it implies close of fds)
```

Comment:
Since recently it is not true always. You can specify a flag not to close the descriptor.

Reply:
Fixed